operator/builtin/output/elastic/elastic.go

```diff
@@ -125,11 +126,44 @@ func (e *ElasticOutput) Stop() error {
 	e.cancel()
 	e.wg.Wait()
 	e.flusher.Stop()
-	// TODO handle buffer close entries
-	_, err := e.buffer.Close()
+
+	entries, err := e.buffer.Close()
+	if err != nil {
+		e.Errorf("Failed to retreive entries")
```

Comment:
Is there just an `e.Error`? The `Errorf` indicates there's some formatting in the string.

Reply:
You mean I should just return that error instead?